src/main/java/net/neoforged/neoforge/common/data/LanguageProvider.java

```diff
@@ -133,4 +136,8 @@ public void add(String key, String value) {
         if (data.put(key, value) != null)
             throw new IllegalStateException("Duplicate translation key " + key);
     }
+
+    public void add(ResourceKey<Level> dimension, String value) {
```

Comment:
This should have a distinct name (i.e. `addDimension()`) as overloads which only change generics are not valid

Reply:
👍